helm/fuel-block-committer/Chart.yaml

```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.10.9
+version: 0.10.10
```

Comment:
Skipping a few versions in order to get the chart and app version to again move in step. Makes it easier to reason about what chart is compatible with what app version.